airflow/config_templates/config.yml

```diff
@@ -1296,6 +1296,15 @@ secrets:
       sensitive: true
       example: ~
       default: ""
+    backends_order:
+      description: |
+        Comma-separated list of secret backends. These backends will be used in the order they are specified.
+        Please note that the `environment_variable` and `metastore` are required values and cannot be removed
+        from the list. Supported values are: `custom`, `environment_variable`, `metastore`.
+      version_added: 2.10.5
```

Comment:
This can not be in 2.10.5 nor in 2.11 it's a new feature. Will have to wait for Airflow 3.

Reply:
Thanks for pointing that out. Changed to the version 3.0.0.